buildenvs/myself.Dockerfile

```diff
@@ -14,14 +14,28 @@ ARG GO_VERSION=${GO_VERSION}
 RUN set -xe; \
     apt-get update; \
     apt-get install -y --no-install-recommends \
-      build-essential \
+      build-essential=12.9 \
       cmake \
-      make \
+      make=4.3-4.1 \
       git; \
+    apt-get clean; \
     go install mvdan.cc/gofumpt@latest; \
     git config --global --add safe.directory /go/src/kraftkit.sh;
 
 WORKDIR /go/src/kraftkit.sh
 
+COPY . .
+
 ENV GOROOT=/usr/local/go
 ENV PATH=$PATH:/go/src/kraftkit.sh/dist
+ENV DOCKER=
+
+# Build the binary
+RUN set -xe; \
+    make kraft; \
+    kraft -h;
+
+# Pull manifests
+RUN set -xe; \
+    kraft --log-level debug --log-type basic pkg source https://github.com/unikraft/app-*; \
+    kraft --log-level debug --log-type basic pkg update;
```

Comment:
This should be removed, since it won't actually be used. The `myself` container is used to build and carry the binary artefacts from the KraftKit project, whereas the `base` container is used to actually build and run unikernels. This code should be migrated to the `base` container:
```diff
- # Pull manifests
- RUN set -xe; \
-     kraft --log-level debug --log-type basic pkg source https://github.com/unikraft/app-*; \
-     kraft --log-level debug --log-type basic pkg update;
```